app/services/redistributed_case.rb

```diff
@@ -30,16 +28,13 @@ def allow!
   end
 
   def ok_to_redistribute?
-    # redistribute if there are no relevant tasks
-    return true if legacy_appeal_relevant_tasks.blank?
-
-    # do not redistribute if any relevant task is open (not completed or cancelled)
+    # If there are open tasks, this case should not be at a judge
+    # waiting for a decision or in case storage waiting to be distributed
+    # it should be "in" caseflow.
     return false if legacy_appeal_relevant_tasks.any?(&:open?)
+    # redistribute if there are either no relevant tasks or if they are completed or cancelled
+    return true if legacy_appeal_relevant_tasks.blank? || legacy_appeal_relevant_tasks.all?(&:closed?)
```

Comment:
```suggestion
    return true if legacy_appeal_relevant_tasks.all?(&:closed?)
```
`all?` returns `true` on an empty array.